src/feedbacks/loading/data-loader/PDataLoader.vue

```diff
@@ -132,15 +133,19 @@ export default defineComponent<Props>({
             dataLoadOccurred: false,
             showEmptyCase: computed(() => !props.disableEmptyCase && state.isEmpty),
             isEmpty: computed(() => {
-                if (!props.data) return false;
+                // if data is null or undefined, it regards as data is not loaded yet. so it's not empty case.
+                if (props.data === null || props.data === undefined) return false;
+
+                // in other cases, it checks if the data is empty.
                 if (Array.isArray(props.data)) return props.data.length === 0;
-                if (typeof props.data === 'boolean') return props.data;
+                if (typeof props.data === 'boolean') return !props.data;
```

Comment:
This code looks like it could cause some side effect. Is it ok?

Reply:
What kind of side effect are you concerning?

Reply:
On logic, the changed code looks normal.
However, there has been no problem with the original code so far, so I'm worried if there are any errors in other parts of this file.

Reply:
Oh I got it. I found NO case that giving boolean value to the data prop, so it will be alright! :)